com.unity.build-report-inspector/Editor/csc.rsp

```diff
@@ -0,0 +1,2 @@
+-r:System.IO.Compression.FileSystem.dll
```

Comment:
Is it necessary to have these **csc.rsp** files in the package?

Reply:
Yes, otherwise it is not possible to use the C# ZipFile class, which is used for displaying apk/app contents.